source/statements.tex

```diff
@@ -921,6 +921,10 @@
 The expression \placeholder{p}\tcode{.return_void()}
 shall be a prvalue of type \keyword{void}.
 \end{itemize}
+\begin{note}
+The overload resolution associated with a \tcode{co_return} statement is affected
+by whether the operand is an implicitly movable entity\iref{class.copy.elision}.
```

Comment:
This is no longer in class.copy.elision, I think.

Reply:
@Quuxplusone PTAL?

Reply:
Jens was right; but I believe I've fixed this in the update 2 months ago.